packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1956,6 +1934,22 @@ export class TransactionController extends BaseController<
     });
   }
 
+  /**
+   * @deprecated No longer used. Kept only to avoid breaking changes. It now performs no operations.
+   * @param transactionID - The ID of the transaction to update.
+   * @param _currentSendFlowHistoryLength - The length of the current sendFlowHistory array.
+   * @param _sendFlowHistoryToAdd - The sendFlowHistory entries to add.
+   * @returns The transactionMeta.
+   */
+  updateTransactionSendFlowHistory(
+    transactionID: string,
+    _currentSendFlowHistoryLength: number,
+    _sendFlowHistoryToAdd: SendFlowHistoryEntry[],
+  ): TransactionMeta {
+    // Return the transaction unchanged
+    return this.#getTransaction(transactionID) as TransactionMeta;
```

Comment:
Very minor, could use `getTransactionOrThrow` to avoid the need to cast.

Reply:
Done https://github.com/MetaMask/core/pull/7326/commits/05f7fd8f93cd2054a1415f7b87f06446d329e1c9